pkg/services/oracle/neofs/neofs.go

```diff
@@ -94,11 +95,13 @@ func parseNeoFSURL(u *url.URL) (*object.Address, []string, error) {
 }
 
 func getPayload(ctx context.Context, c client.Client, addr *object.Address) ([]byte, error) {
-	obj, err := c.GetObject(ctx, new(client.GetObjectParams).WithAddress(addr))
+	res, err := c.GetObject(ctx, new(client.GetObjectParams).WithAddress(addr))
```

Comment:
So it returns both `err` and `res.Status()`?

Reply:
Yes: `err` carries internal client failures only, while `res.Status()` - server's response statuses. Each application is expected to handle error handling by itself, while the client returns a response from the server untouched.